docs/pages/iconbutton.js

```diff
@@ -197,26 +191,32 @@ function SectionsIconButtonDropdownExample() {
             description:
               'Indicates that a component controls the appearance of interactive popup elements, such as menu or dialog. See the [Accessibility](#ARIA-attributes) guidelines for details on proper usage.',
           },
+          {
+            name: 'role',
+            type: 'button',
+            description:
+              'Sets button interaction in the component. See the [role](#Role) variant to learn more.',
+          },
           {
             name: 'selected',
             type: 'boolean',
             description:
               'Toggles between binary states: on/off, selected/unselected, open/closed. See the [selected](#Selected-state) variant to learn more.',
           },
+          {
+            name: 'type',
```

Comment:
This is the new part, the other changes to this file are alphabetization and cleanup